packages/core/package.json

```diff
@@ -18,20 +18,43 @@
     "functional"
   ],
   "ava": {
+    "babel": {
+      "plugins": [
+        [
+          "transform-object-rest-spread",
```

Comment:
I thought ava included this by default? https://github.com/avajs/ava#es2017-support